### PR TITLE
10 l10n br migration

### PR DESCRIPTION
**Pull request by renatonlima**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mileo** [comment]
:+1:

File: addons/l10n_br/models/account.py [hunk 1]
```diff
@@ -4,12 +4,20 @@
...
+
+
 class AccountTaxTemplate(models.Model):
     """ Add fields used to define some brazilian taxes """
     _inherit = 'account.tax.template'
 
     tax_discount = fields.Boolean(string='Discount this Tax in Prince',
-                                    help="Mark it for (ICMS, PIS e etc.).")
+                                  related='tax_group_id.tax_discount',
+			          help="Mark it for (ICMS, PIS e etc.).")
```
Comment (leorochael) [inline addons/l10n_br/models/account.py]: There are TAB characters embedded in this line.

File: addons/l10n_br/models/account.py [hunk 2]
```diff
@@ -25,7 +33,8 @@ class AccountTax(models.Model):
     """ Add fields used to define some brazilian taxes """
     _inherit = 'account.tax'
 
-    tax_discount = fields.Boolean(string='Discount this Tax in Prince', 
+    tax_discount = fields.Boolean(string='Discount this Tax in Prince',
+			          related='tax_group_id.tax_discount',
```
Comment (leorochael) [inline addons/l10n_br/models/account.py]: There are tabs embedded in this line as well

**Comment by mart-e** [comment]
Hello,

Thank you for you patch but we can not add new fields [in a stable version](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean).
Could you detail what your changes are?

Also it seems that you have signe the Akretion CLA with your address `@akretion.com` while you are committing with your `@akretion.com.br` address.
Could you change it so we can integrate your patch?
Regards

File: addons/l10n_br/models/account.py [hunk 3]
```diff
@@ -4,12 +4,20 @@
 from odoo import fields, models
 
 
+class AccountTaxGroup(models.Model):
+    _inherit = 'account.tax.group'
+
+    tax_discount = fields.Boolean(string='Discount this Tax in Prince',
```
Comment (leorochael) [inline addons/l10n_br/models/account.py]: `in Prince` ➡️ `from the Price`. This whole sentence deserves better phrasing in English (and in AccountTaxTemplate as well.

Comment (mileo) [inline addons/l10n_br/models/account.py]: :+1:

**Comment by tivisse** [comment]
@renatonlima Sorry but we cannot add fields in a stable version, as said by @mart-e.

I close this PR.

**Pull request closed without merging**